packages/web/src/components/leaderboard/leaderboard-table-row/LeaderboardTableRow.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  HoverOnBgSection,
+  HoverSection,
+  TableColumn,
+  Wrapper,
+  WrapperHoverBackground,
+} from "./LeaderboardTableRow.styles";
+import PointComposition from "../point-composition/PointComposition";
+import UserColumn from "../user-column/UserColumn";
+import { LeaderModel } from "@models/leaderboard/leader-model";
+
+const LeaderboardTableRow = ({
+  item,
+  tdWidths,
+  isMobile,
+  me = false,
+}: {
+  item: LeaderModel;
+  tdWidths: number[];
+  isMobile: boolean;
+  me?: boolean;
+}) => {
+  const {
+    rank,
+
+    hide,
+
+    formattedAddress,
+    address,
+
+    swapVolume,
+    positionValue,
+    stakingValue,
+
+    pointSum,
+    swapFeePoint,
+    poolRewardPoint,
+    stakingRewardPoint,
+    referralRewardPoint,
+  } = item;
+
+  const Hover = me ? HoverSection : HoverOnBgSection;
+  const TableWrapper = me ? WrapperHoverBackground : Wrapper;
```

Comment:
Is there a reason you didn't pass the styled component as props ?

Reply:
In this code, which styled components are going to be used depends on the `me` variable. and this component takes `me` as a prop from parent, so I did not see the need to pass those styled components from the parent since parent can still have a control over/can configure which styled component to use in the child component(`LeaderboardTableRow`) by using the `me` prop without actually importing the child component(`LeaderboardTableRow`)-specific styled components.

I also thought, it would make it easier for me to write any testcodes later on as well.

Reply:
I understand, I see it's inherited and implemented.